rwkv.cpp

```diff
@@ -227,7 +232,7 @@ struct rwkv_context * rwkv_init_from_file(const char * file_path, const uint32_t
     );
 
     RWKV_ASSERT_NULL(
-        model->data_type != 6,
+        model->data_type != 5,
```

Comment:
This is Q4_2, which was removed last week, but you should change the warning text to say Q4_2 and also keep one for Q4_3 as well which is still unsupported

Reply:
will fix them all